### PR TITLE
add time namespaces

### PR DESCRIPTION
**Pull request by lengrongfu**
issue: https://github.com/containers/oci-spec-rs/issues/121

### CODE REVIEW

File: src/runtime/linux.rs [hunk 1]
```diff
@@ -752,6 +752,9 @@ pub enum LinuxNamespaceType {
 
     /// Network Namespace for isolating network devices, ports, stacks etc.
     Network = 0x40000000,
+
+    /// Time Namespace for isolating the clocks
+    Time = 0x80000000,
```
Comment (saschagrunert) [inline src/runtime/linux.rs:757]: This should be `0x00000080` according to:
https://github.com/torvalds/linux/blob/0983f6bf2bfc/tools/testing/selftests/pidfd/pidfd.h#L26

**Review by saschagrunert — APPROVED** [review]

**Comment by saschagrunert** [comment]
@Furisto @utam0k PTAL

**Comment by utam0k** [comment]
@lengrongfu May I ask you to check the error from lint?

**Comment by saschagrunert** [comment]
> @lengrongfu May I ask you to check the error from lint?

I addressed them in #123

**Comment by lengrongfu** [comment]
> @lengrongfu May I ask you to check the error from lint?

It seems that this lint error is a problem that existed before, and I can submit a new pr to fix it.

**Comment by utam0k** [comment]
@lengrongfu Could you rebase this brach from the main branch?

**Comment by lengrongfu** [comment]
> @lengrongfu Could you rebase this brach from the main branch?

done.

**Pull request merged**